src/frontend/js/utils/test/factories/joanie.ts

```diff
@@ -81,6 +81,7 @@ export const EnrollmentFactory = factory((): Enrollment => {
     was_created_by_order: false,
     created_on: faker.date.past({ years: 1 }).toISOString(),
     orders: [],
+    certificate_id: faker.string.uuid(),
```

Comment:
certificate_id is optional, from my point of view, by default the value should be `null`.